x-pack/plugin/watcher/src/internalClusterTest/java/org/elasticsearch/xpack/watcher/test/integration/HistoryIntegrationTests.java

```diff
@@ -104,10 +103,7 @@ public void testFailedInputResultWithDotsInFieldNameGetsStored() throws Exceptio
 
         new ExecuteWatchRequestBuilder(client()).setId("test_watch").setRecordExecution(true).get();
 
-        assertBusy(() -> {
-            refresh(".watcher-history*");
-            assertHitCount(prepareSearch(".watcher-history*").setSize(0), 1);
```

Comment:
`.setSize(...)` drops out here (and a few other places) -- is that okay?

Reply:
Some places only needed the count (so size=0 was fine), and some wanted the single record (so size=1 was fine). Since we always expect either 0 or 1 documents in this index it didn't seem to cause any problems to just use the default size. Worst case, we pull back a single history document that we don't care about that gets immediately garbage collected. It avoided passing an argument into this method that might just add confusion.